static/css/components/header-bar--desktop.less

```diff
@@ -34,7 +34,6 @@
     -webkit-box-ordinal-group: 3;
     -ms-flex-order: 2;
     order: 2;
-    min-width: 250px;
```

Comment:
Note to self: in IE11 the min-width causes the search to shrink like so:
<img width="888" alt="Screen Shot 2021-01-07 at 5 25 37 PM" src="https://user-images.githubusercontent.com/148752/103963114-5ce70200-510d-11eb-88e3-4928813ed271.png">

Without it:
<img width="920" alt="Screen Shot 2021-01-07 at 5 26 08 PM" src="https://user-images.githubusercontent.com/148752/103963206-8d2ea080-510d-11eb-843c-c549088183fc.png">


The min-width doesn't seem to be doing anything in Chrome or Firefox since we are using flexbox so I think it's safe to remove.